### ISSUE TITLE
AUR package

### ISSUE THREAD

**Issue by MiltFra** [notable/notable#74]
I would be great if there was an AUR Package! There's not much to say about that.

**Comment by fabiospampinato** [comment]
@MiltFra is that used only by Arch?

**Comment by Philiphlop** [comment]
> @MiltFra is that used only by Arch?

AUR = Arch User Repository  https://wiki.archlinux.org/index.php/Arch_User_Repository. Though it is used by other OS it's not as widespread as Debian etc, these are often Arch spin-offs. 

AppImage, Snap, and FlatPak is supported on Arch.

**Comment by MiltFra** [comment]
> AppImage, Snap, and FlatPak is supported on Arch.

While this is true, I doubt that they are prominent here, at least I haven't had to worry about that. Basically everything is in the AUR. 
I don't know how tedious it is to setup an AUR package since I haven't actually done it. So I just wanted to ask if it was possible.

**Comment by Thrilleratplay** [comment]
@MiltFra  After failing to get the snap and appimage packages to work, I tried to create a AUR package found [here](https://aur.archlinux.org/packages/notable-bin/)

**Comment by MiltFra** [comment]
@Thrilleratplay It seems to be working, thank you!

**Comment by Thrilleratplay** [comment]
@fabiospampinato Maintenance at this point should be rather easy as all I am doing is extracting the contents Debian .deb package. This is my first AUR so these instructions may be have some issues.

  1. Create an account on [Arch's AUR page](https://aur.archlinux.org/).  Be sure to add a public SSH key.
  2. Let me know what your user name is and I will add you as a co-maintainer to the package.
  3. Clone the repo `git clone ssh://aur@aur.archlinux.org/notable-bin.git` which will contain a few files but the key one is [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=notable-bin)

To verify a AUR you will probably need a Arch based system with AUR either through Docker or a VM.

When updating:
* If the AUR package is being updated, such as adding yourself as a maintainer, increment the `pkgrel`
* If Notable is being updated, set `pgkrel=1`, update the `pkgver` to the Notable version and update the MD5 and SHA512 to be checksums of the new Debian package. 

```bash
# verify package and checksums
makepkg --verifysource --check -Cc  

# Update .SRCINFO
makepkg --printsrcinfo > .SRCINFO

# Update package repo
git add PKGBUILD .SRCINFO 

git commit -m "some commit message"

git push
```

**Comment by fabiospampinato** [comment]
@Thrilleratplay thanks a lot, it seems a bit laborious so let me focus on getting v1.1.0 ready first.

**Comment by Thrilleratplay** [comment]
@fabiospampinato I prefer you focus on that too.  I will do my best to keep the AUR package up to date in the meantime.

**Comment by fabiospampinato** [comment]
@Thrilleratplay it took a while but I'm now registered on the AUR system, my username is `fabiospampinato`, could you add me as a co-maintainer?

**Comment by fabiospampinato** [comment]
@Thrilleratplay The whole process seems pretty straightforward, and most importantly `makepkg` can be used under macOS it seems, so I won't have to jump into an Arch VM each time to manage this.

I'd say I can take care of this in the future and make sure the arch package is updated whenever I push a new release.

**Comment by Thrilleratplay** [comment]
@fabiospampinato   Currently the maintainer is `kugland`, if logged into AUR, [his account has his email address](https://aur.archlinux.org/account/kugland).  The build process I used hacked up the Debian package to install it through AUR.  I orphaned it after finding out Notable releases included a pacman package, which is what Arch uses.  It would be much easier and saner to use the generated pacman package as a community release instead of through AUR.  I am not sure what the requirements are for that. Let me try to find out.
